cmd/cdi-importer/importer.go

```diff
@@ -219,3 +211,12 @@ func main() {
 	}
 	klog.V(1).Infoln(message)
 }
+
+func convertQuantityToInt64(size resource.Quantity) int64 {
```

Comment:
If this is truly needed it smells like it should be a shared utility function.  This sort of conversion seems like a pretty common operation.  Was this code required for those other cases?  If not, why not?  If so, can we share that logic with this case too?

Reply:
Indeed, there is a function for that. I don't know how could I have missed it. But I did. Fixing...